app/controllers/cinema_halls_controller.rb

```diff
@@ -1,26 +1,20 @@
 class CinemaHallsController < ApplicationController
-  
+  skip_before_action :doorkeeper_authorize!, only: %i[index show]
+
   def index
-    if Repository::TicketDeskRepository.new.exists?(id: params[:ticket_desk_id])
-      render json: CinemaHalls::Representer.new(CinemaHall.all).single.order(volume: :asc) , except: [:created_at, :updated_at]
-    end
+      render json: CinemaHall.all.order(volume: :asc) , except: [:created_at, :updated_at]
   end
 
   def show
   end
 
   def create
+    #if employer?
```

Comment:
Don't keep commented code in the repository, please.
https://kentcdodds.com/blog/please-dont-commit-commented-out-code